.circleci/normalize_yaml_fragment.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import sys
+import yaml
+import cimodel.lib.miniyaml as miniyaml
+
+
+USE_PYYAML_FORMATTER = False
+
+
+def regurgitate(depth):
```

Comment:
Please don't use global variables when you don't have to
```suggestion

def regurgitate(depth, USE_PYYAML_FORMATTER = False):
```

Reply:
fixing